core-ui/src/state/navigation/addAdditionalNodes.ts

```diff
@@ -0,0 +1,56 @@
+import { ConfigFeatureList, NavNode, Scope } from '../types';
+import { isEqual } from 'lodash';
+import { mapBusolaResourceToNavNode } from '../resourceList/mapBusolaResourceToNavNode';
+
+import { createClusterNode } from 'components/Clusters/views/ClusterOverview/clusterDetailsNode';
+import { extensionsNavNode } from 'components/BusolaExtensions/extensionsNode';
+import { helmReleasesNode } from 'components/HelmReleases/helmReleasesNode';
+import { createCustomResourcesNavNode } from 'components/CustomResources/customResourcesNode';
+import { namespaceOverviewNode } from 'resources/Namespaces/namespaceOverviewNode';
+
+import * as secretMetadata from 'resources/Secrets';
+import * as crdMetadata from 'resources/CustomResourceDefinitions';
+import * as cmMetadata from 'resources/ConfigMaps';
+
+export const addAdditionalNodes = (
+  navNodes: NavNode[],
+  scope: Scope,
+  configFeatures: ConfigFeatureList,
+) => {
+  const extNavList = [...navNodes];
+
+  addResource(createClusterNode(scope), extNavList.length, extNavList);
```

Comment:
I'm not sure about this - I'd go for 2 separate constant nodes (one for Cluster Details, one for navigating back to clusterwide) instead of a function - they don't differ a lot now, but we'll probably add a navigation handler later.

Reply:
agreed